src/components/Image/tests/Image.test.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import Image from '..';
+import { mount } from 'enzyme';
+
```

Comment:
1) Add root level `describe('<Image />', () => {` describe and included all tests inside it.
2) Then add tests for a default/required props case for Image Component. Refer. Chip Component.